dev-db/gigabase/gigabase-3.83-r2.ebuild

```diff
@@ -48,15 +54,6 @@ src_test() {
 	[[ $failcnt != 0 ]] && die "See warnings above for tests that failed"
```

Comment:
rewrite `src_test`, keeping it more idiomatic: `die` on the first failure, instead of counting the number of failures. Don't forget to make the loop variable `local` too!